share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -98,3 +108,38 @@ macro(rocm_create_package)
     endif()
     include(CPack)
 endmacro()
+
+
+function (rocm_set_system_name)
```

Comment:
Ok can we have this as `rocm_get_system_name` where the user passes in their own variable? Then the user can decide if they want to set `CMAKE_SYSTEM_NAME` or `CPACK_SYSTEM_NAME` by them setting it manually.

Reply:
I am removing this as I moved the functionality to rocm_create_package. Do you think it is worth keeping as a separate call?

Reply:
If you dont think its useful outside of `rocm_create_package` then just remove it.